sbt/src/sbt-test/project/unified/test

```diff
@@ -0,0 +1,13 @@
+> root/Compile/compile
+
+# "Global" now works in shell. optional whitespace arond /
+> show Global / cancelable
+
+# "ThisBuild" now works in shell. optional whitespace arond /
```

Comment:
The optional whitespace around `/` should be tested for every example.
  
Also, typo in "arond".